### PR TITLE
[DM-1005] Document LwM2M 1.1 features

### PR DESCRIPTION
**Pull request by bggov**

### CODE REVIEW

File: content/protocol-integration/lwm2m-bundle/overview.md [hunk 1]
```diff
@@ -7,11 +7,17 @@ layout: redirect
...
+
+You can connect any device supporting LWM2M 1.1 or LWM2M 1.0 to {{< product-c8y-iot >}} without programming. We expect the device and its capabilities (such as firmware update) to be compliant to the LWM2M specification. The device must support the UDP binding of the LWM2M standard. There is some possibility to connect also a devices using CoAP over TCP binding, but this is still in an experimental phase and not fully supported. That's why connecting LWM2M devices using CoAP over TCP is not recommended
+{{< /c8y-admon-info >}}
+
+{{< c8y-admon-info >}}
+{{< product-c8y-iot >}} LWM2M 1.1 introduces a new client's side "send" operation. If a sensor or number of sensors want to send measurements to the server, they can use the "send" operation to send single or composite measurement data. LWM2M 1.1 "send" operation makes this simple on protocol level, compared to LWM2M 1.0 where the level of complexity is higher using MQTT protocol for these purposes.
```
[the remark this answers is not in the document]
Comment (bggov) [reply content/protocol-integration/lwm2m-bundle/overview.md:20]: From customer's side it's good to know that we're already supporting this in a much simple way. I agree that I'm getting in too many technical details, so I can remove some of them. I can leave only the first part, where it says we support 'send', but remove the rest of the details.

File: content/protocol-integration/lwm2m-bundle/register-device.md [hunk 2]
```diff
@@ -259,7 +259,8 @@ See the table below for information on additional fields:
 <tr style="height: 26px;">
 <td style="text-align: left">Binding mode / bindingMode</td>
 <td style="text-align: left">String</td>
-<td style="text-align: left">The LWM2M binding mode to be reported to the device. Supported are "UQ" (default, queuing) and "U" (unqueued). Note that {{< product-c8y-iot >}} will always queue operations.</td>
+<td style="text-align: left">The LWM2M binding mode to be reported to the device. For LWM2M 1.0 devices supported modes are "UQ" (default, queuing) and "U" (unqueued).
+Note, that since {{< product-c8y-iot >}} LWM2M 1.1, &ldquo;Q&rdquo; (queue) mode is not supported, and the default mode will be &ldquo;U&rdquo; (unqueued). Although {{< product-c8y-iot >}} will always queue operations.</td>
```
Comment (sag-eweingaertner) [inline content/protocol-integration/lwm2m-bundle/register-device.md:263]: I'd rephrase this a little: {{< product-c8y-iot >}} will always queue operations regardless if the device is connected or not, hence this setting has virtually zero effect on the behavior of {{< product-c8y-iot >}}.

**Review by sag-eweingaertner — APPROVED** [review]

**Review by rssag — APPROVED** [review]
language review as commit + approval

File: content/protocol-integration/lwm2m-bundle/overview.md [hunk 3]
```diff
@@ -7,11 +7,17 @@ layout: redirect
...
+{{< product-c8y-iot >}} LWM2M 1.1 is backward compatible. What has been working on {{< product-c8y-iot >}} LWM2M 1.0 will also be working on {{< product-c8y-iot >}} LWM2M 1.1. However, {{< product-c8y-iot >}} LWM2M 1.1 introduces some new features which are not compatible with LWM2M 1.0 devices. If you try to run some of these features with LWM2M 1.0 devices, you may receive an error message response.
+
+You can connect any device supporting LWM2M 1.1 or LWM2M 1.0 to {{< product-c8y-iot >}} without programming. {{< product-c8y-iot >}} expects the device and its capabilities (such as firmware update) to be compliant to the LWM2M specification. The device must support the UDP binding of the LWM2M standard. There is also the possibility to connect a device using CoAP over TCP binding, but this is still in an experimental phase and not fully supported. That's why connecting LWM2M devices using CoAP over TCP is not recommended
```
Comment (sag-eweingaertner) [inline content/protocol-integration/lwm2m-bundle/overview.md:16]: I missed this before. Please do not mention TCP or any other binding mode at all. Please simply write above (see second marker) only that we support CoAP-UDP.

File: content/protocol-integration/lwm2m-bundle/overview.md [hunk 4]
```diff
@@ -7,11 +7,17 @@ layout: redirect
 Lightweight M2M (LWM2M) is a traffic and resource-optimized protocol to remotely manage IoT devices. The protocol is standardized by the Open Mobile Alliance. For more information, see [http://openmobilealliance.org/iot/lightweight-m2m-lwm2m](http://openmobilealliance.org/iot/lightweight-m2m-lwm2m).
 
 {{< c8y-admon-important >}}
-{{< product-c8y-iot >}} currently supports LWM2M 1.0.
+{{< product-c8y-iot >}} currently supports LWM2M 1.1.
```
Comment (sag-eweingaertner) [inline content/protocol-integration/lwm2m-bundle/overview.md:10]: -> {{< product-c8y-iot >}} currently supports LWM2M 1.1 over CoAP and UDP.

**Review by sag-eweingaertner — CHANGES_REQUESTED** [review]

**Review by Oezge-Akin — APPROVED** [review]

**Review by sag-eweingaertner — APPROVED** [review]

File: content/protocol-integration/lwm2m-bundle/shell-commands.md [hunk 5]
```diff
@@ -57,60 +57,87 @@ In the next table you will see all available commands and a brief description of
...
 <td align="left">write /&lt;objectID&gt;/&lt;instanceID&gt;/&lt;resourceID&gt; &lt;value&gt;</td>
+<td align="center">1.0, 1.1</td>
 <td align="left">Writes value to a resource on the device.</td>
 </tr>
 <tr>
-<td align="left">cancelobservation /&lt;objectID&gt;/&lt;instanceID&gt;/&lt;resourceID&gt;</td>
-<td align="left">Cancels the observation functionality from the desired resource.</td>
+<<td align="left">cwrite /&lt;objectID&gt;/&lt;instanceID&gt;/&lt;resourceID&gt; &lt;value&gt; [/&lt;objectID&gt;/&lt;instanceID&gt;/&lt;resourceID&gt; &lt;value&gt;]</td>
+<td align="center">1.1</td>
+<td align="left">Composite writes of one or more values to a resources on the device. The data will be written to the listed resource paths in a single request</td>
```
Comment (BeateRixen) [inline content/protocol-integration/lwm2m-bundle/shell-commands.md:103]:
```suggestion
<td align="left">Composite writes of one or more values to a resource on the device. The data will be written to the listed resource paths in a single request</td>
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@bggov Can this be merged, and is this for 10.15 only?
